### DISCUSSION TITLE
Playing Video on Low Power Mode.

### DISCUSSION THREAD

**Discussion by developersaintt** [videojs/video.js#7702] [Q&A]
Hi there,

I have been wondering if there is any way to play the video in the low power mode on iOS because right now we cannot play any video while the device is on low power mode.

Thanks for the help.

**Comment by hartman** [comment]
Are these autoplay videos ? Because autoplay is not allowed in low power mode.

**Reply by developersaintt** [reply]
Yes that's what I was wondering if videojs can also play the video in low-power mode on iOS...
